tasks/rke2.yml

```diff
@@ -142,11 +142,11 @@
 - name: Copy Custom Manifests
   ansible.builtin.template:
     src: "{{ item }}"
-    dest: "{{ rke2_data_path }}/server/manifests/"
+    dest: "{{ rke2_data_path }}/server/manifests/{{ item | basename | replace('.j2', '')}}"
     owner: root
     group: root
-    mode: 0644
-  with_items: "{{ rke2_custom_manifests }}"
+    mode: 0640
+  with_fileglob: "{{ rke2_custom_manifests }}/*"
```

Comment:
I thin't it's better to keep it as a list of manifests. They can be in different folders or subfolders

Reply:
Makes sense. I guess you could have `{{ query('fileglob', rke2_custom_manifests + '/*') }}` as an example instead.

Reply:
well, i think this is outdated

https://github.com/lablabs/ansible-role-rke2/blob/859a27f112add823280e9609a2c0056c0f30247f/tasks/rke2.yml#L252-L259

Reply:
> well, i think this is outdated
> 
> https://github.com/lablabs/ansible-role-rke2/blob/859a27f112add823280e9609a2c0056c0f30247f/tasks/rke2.yml#L252-L259

Yep - with the same code I wrote! So now it includes the glob functionality that @MonolithProjects said he didn't want to implement. Was that on purpose?

Reply:
No @simonfelding, i overlooked. Seems like you included the code in #260 . Will check it tomorrow as it is probably breaking the Custom Manifests (#280)